std/zip.d

```diff
@@ -420,22 +546,26 @@ public:
         assert(de._compressedData.length == de._compressedSize, "Archive member compressed failed.");
     }
 
-    /** Delete de from the archive.
+    /**
+     * Delete ArchiveMember de from the archive. Uses the name of the member to detect which element to delete.
```

Comment:
Preferably keep lines under 80-100 characters.

Reply:
I read somewhere in the docs, that they should be less than 120, but 100 or 80 as a limit is fine for me too. :-)

Reply:
120 is a hard limit, where even "ugly" methods of breaking up a line should be used. There is nothing ugly about wrapping a paragraph of text at 80 characters instead of 120, though.